src/main/java/toast/specialMobs/entity/enderman/EntityBlindingEnderman.java

```diff
@@ -24,7 +25,9 @@ public EntityBlindingEnderman(World world) {
     /// Called every tick while this entity is alive.
     @Override
     public void onLivingUpdate() {
-        if (this.entityToAttack != null && this.entityToAttack instanceof EntityLivingBase && this.getDistanceSqToEntity(this.entityToAttack) < 256.0) {
+        if (this.entityToAttack != null && this.entityToAttack instanceof EntityLivingBase && !(this.entityToAttack instanceof FakePlayer) 
```

Comment:
Redundant nullcheck and `instanceof`.

When object is an instance of a class, it is also not null:
:
```java
this.entityToAttack != null && this.entityToAttack instanceof EntityLivingBase 
```
→
```java
this.entityToAttack instanceof EntityLivingBase 
```